Library/.rubocop.yml

```diff
@@ -123,6 +123,10 @@ Style/Documentation:
 Style/Encoding:
   Enabled: true
 
+# disabled until we get the Metrics/LineLength down to 80.
```

Comment:
Is that something we're actively working on/making progress in?

Reply:
Not currently.